apis/r/tests/testthat/test-SOMADataFrame.R

```diff
@@ -820,3 +820,28 @@ test_that("factor levels can grow without overlap", {
     ref <- rbind( tibble::as_tibble(tbl_1), tibble::as_tibble(tbl_2) )
     expect_equal(tbl, ref)
 })
+
+test_that("factor levels cannot beyond index limit", {
```

Comment:
```suggestion
test_that("factor levels cannot extend beyond index limit", {
```

Reply:
Good catch.  Will add that as a quick `[cp skip]` commit!

Reply:
Done.